src/server/api/routers/post.ts

```diff
@@ -46,7 +48,23 @@ export const postRouter = createTRPCRouter({
             // query: z.function().args().returns(z.string())
         }))
         .query(async ({input}) => {
-            const queryURI = encodeURI(input.query);
+            console.log("SEARCH: " + input.query)
```

Comment:
Do we still need this?

Reply:
That was for testing purposes, this can be removed?